index.html

```diff
@@ -1,13 +1,18 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,1,0" />
-        <title>🛋️ CushyStudio</title>
-    </head>
-    <body>
-        <div id="root"></div>
-        <script type="module" src="/src/front/ui/main.tsx"></script>
-    </body>
-</html>
+
+<head>
+    <script src="http://localhost:8097"></script>
```

Comment:
what is this added thing ?

Reply:
Ah that, that is used for the react dev tools. When they are not running nothing should happen.